inst/plumber/v1/endpoints.R

```diff
@@ -281,22 +270,24 @@ function(req) {
 #* @param welfare_type:[chr] Welfare Type. Options are "income" or "consumption"
 #* @param reporting_level:[chr] Reporting level. Options are "national", "urban", "rural".
 #* @param ppp:[dbl] Custom Purchase Power Parity (PPP) value.
-#* @param version:[chr] Data version. Defaults to most recent version. See api/v1/versions for all available versions
+#* @param release_version:[chr] date when the data was published in YYYYMMDD format
+#* @param ppp_version:[chr] ppp year to be used
+#* @param version:[chr] Data version. Defaults to most recent version. See api/v1/versions
+#* @param identity:[chr] One of "PROD" (production) (default), "INT" (internal) and "TEST"
 #* @param format:[chr] Response format. Options are "json", "csv", or "rds".
 #* @serializer switch
 function(req) {
   # Process request
-  # browser()
   params <- req$argsQuery
   params$lkup <- lkups$versions_paths[[params$version]]
   params$format <- NULL
   params$version <- NULL
 
   # Define default arguments
-  # if (is.null(params$country))
-  #   params$country <- "all"
-  # if (is.null(params$year))
-  #   params$year <- "all"
+  if (is.null(params$country))
+    params$country <- "all"
+  if (is.null(params$year))
+    params$year <- "all"
```

Comment:
Ok, so this handled in one of the filters and can therefore be removed.
![image](https://user-images.githubusercontent.com/1552634/185366961-c327b433-2d3d-4500-9e76-aa97fa74fb7d.png)